src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -68,6 +72,8 @@
 {
     private static final String HTTP_TYPE_NAME = "http";
 
+    private static final StringFW EVENT_TYPE_CHALLENGE = new StringFW("challenge");
```

Comment:
Please move this to non-static and drive the name from `SseConfiguration` defaulting to `"challenge"`.

In cases where an upstream `SSE` server already uses a `"challenge"` event with different semantics, we need to be able to reconfigure this logic to use a non-colliding name.